src/main/java/org/tailormap/api/configuration/TemporaryTokenCleanupService.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2025 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package org.tailormap.api.configuration;
+
+import java.lang.invoke.MethodHandles;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.tailormap.api.repository.TemporaryTokenRepository;
+
+@Service
+@Transactional
+public class TemporaryTokenCleanupService {
+  private static final Logger logger =
+      LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  @Scheduled(fixedDelay = 1, timeUnit = TimeUnit.MINUTES, initialDelay = 1)
+  public void cleanExpiredTokens(TemporaryTokenRepository temporaryTokenRepository) {
```

Comment:
this does not work/is not allowed, yet works in the IDE:
`BeanCreationException: Error creating bean with name 'temporaryTokenCleanupService' defined in URL [jar:nested:/home/spring/tailormap-api.jar/!BOOT-INF/classes/!/org/tailormap/api/configuration/TemporaryTokenCleanupService.class]: Could not create recurring task for @Scheduled method 'cleanExpiredTokens': Only no-arg methods may be annotated with @Scheduled`

(and Autowiring does not work because it does not provide a transaction manager in the started thread)